okapi-core/src/main/java/org/folio/okapi/service/impl/ProcessModuleHandle.java

```diff
@@ -98,6 +92,47 @@ public void start(Handler<AsyncResult<Void>> startFuture) {
     }
   }
 
+  @Override
+  public void onStdout(ByteBuffer buffer, boolean closed) {
+    byte[] bytes = new byte[buffer.remaining()];
+    buffer.get(bytes);
+    String s = new String(bytes);
```

Comment:
This should not use the Charset of the server where Okapi runs but StandardCharsets.UTF_8.

Reply:
ok. done